packages/expo-modules-core/ios/ModuleRegistryAdapter/EXModuleRegistryAdapter.m

```diff
@@ -76,4 +89,13 @@ - (instancetype)initWithModuleRegistryProvider:(EXModuleRegistryProvider *)modul
   return extraModules;
 }
 
+- (nullable SwiftInteropBridge *)swiftInteropBridgeModulesRegistry:(EXModuleRegistry *)moduleRegistry
```

Comment:
`nullable` is the same as `_Nullable`? I have never seen someone using `nullable` outside of the property declaration.